doc/dev/doc/getting-started.md

```diff
@@ -1,5 +1,30 @@
 # Getting Started
 
+## Download and Install
+
```

Comment:
Is the reason of adding this information to this page and not to "download and install" document making it more visible? 

If it is related to download and install, we can directly put it there. I left to you